CHANGELOG.rst

```diff
@@ -1,6 +1,16 @@
 [Unreleased]
 ------------
 
+Fixed
+^^^^^
+* `@lukehsiao`_: The features/labels/gold_label key tables were not properly
```

Comment:
Update this part with solving potential deadlock issue.

Reply:
I was going to do that, but realized that the deadlock issue was only there because of the changes made in this PR. The deadlock issue shouldn't have happened when it was just on_conflict_do_nothing.